tests/cephadm/test_admin_add_labels.py

```diff
@@ -0,0 +1,64 @@
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def _add_host_label(instance):
+    """
+    Method add multiple labels to host
```

Comment:
Host is `cephadmin` so please mention that here i.e 

> Method to add multiple labels to cephadmin node

Reply:
Thanks @sunilangadi2
Updated